### ISSUE TITLE
Can't install re-formality on M1 mac

### ISSUE THREAD

**Issue by dnmct** [shakacode/re-formality#100]
When i try to install re-formality via yarn i get this error message that tells me to file an issue here, so here I am:

```
re-formality-ppx does not support this platform :(

re-formality-ppx comes prepacked as built binaries to avoid large
dependencies at build-time.

If you want re-formality-ppx to support this platform natively,
please open an issue at our repository, linked above. Please
specify that you are on the darwin platform,
on the arm64 architecture.
```

**Comment by alexfedoseev** [comment]
@dnmct Hey @dnmct, thanks for the issue! Unfortunately, I don't have M1 at hand, I need to research what needs to be done to support it. Can't give any ETA though.

**Comment by alexfedoseev** [comment]
Should be fixed in #106. Feel free to reopen if it doesn't work.
